app/src/pages/inside/projectSettingsPageContainer/content/analyzerContainer/indexSettings/indexSettings.jsx

```diff
@@ -27,6 +27,7 @@ import { useTracking } from 'react-tracking';
 import { showModalAction } from 'controllers/modal';
 import { useDispatch } from 'react-redux';
 import { Checkbox } from 'componentLibrary/checkbox';
+import { PROJECT_SETTINGS_ANALYZER_EVENTS } from 'components/main/analytics/events/ga4Events/projectSettingsTab/analyzerTab/indexSettings/indexSettingsEvents';
```

Comment:
This path looks too long, looks like it's time to create a new alias in the webpack config for the events folder